### DISCUSSION TITLE
Replacing Sonoff ZB 3.0 dongle plus by same model problem

### DISCUSSION THREAD

**Discussion by 2dlight** [Koenkk/zigbee2mqtt#19083] [Devices]
Hi,
As I am a newby, this question...
Z2M with my Sonoff zb 3.0 P usb coordinator firmware 221226 is running well, on raspbery pi.
I have a spare Sonoff 3.0 P, flashed with coordinator version 20230507.
After changing to this spare coordinator Sonoff USB Z2M isn't working.
The both usb coordinators are the same hardware (Sonoff p,) same usb port.
What is probably wrong? 
Thanks

**Comment by Matriciel** [comment]
They haven't the same physical address and re-include all device is necessary.
It is also possible to flash the address of first dongle on second (process available on Z2M site).

**Comment by 2dlight** [comment]
Flashed the second ieee adress of my first dongle to my second with TI flash programmer 2.
But no frontend and no Z2M working.
What could be wrong?

**Reply by francisp2** [reply]
What do the logs say ?

**Reply by 2dlight** [reply]
Sorry, as I am new to it, where to find the log?

**Reply by francisp2** [reply]
Depends on how you installed Zigbee2MQTT

On a standard linux install  /opt/zigbee2MQTT/data/log

**Reply by 2dlight** [reply]
My log:

info  2023-09-26 14:48:41: Logging to console and directory: '/opt/zigbee2mqtt/data/log/2023-09-26.14-48-41' filename: log.txt
info  2023-09-26 14:48:41: Starting Zigbee2MQTT version 1.30.2 (commit #cdf62ea)
info  2023-09-26 14:48:41: Starting zigbee-herdsman (0.14.96)
error 2023-09-26 14:48:42: Error while starting zigbee-herdsman
error 2023-09-26 14:48:42: Failed to start zigbee
error 2023-09-26 14:48:42: Check https://www.zigbee2mqtt.io/guide/installation/20_zigbee2mqtt-fails-to-start.html for possible solutions
error 2023-09-26 14:48:42: Exiting...
error 2023-09-26 14:48:42: Error: Error while opening serialport 'Error: Error: No such file or directory, cannot open /dev/serial/by-id/usb-ITead_Sonoff_Zigbee_3.0_USB_Dongle_Plus_2825e0cf8b29ec11a07e6d7840c9ce8d-if00-port0'
    at SerialPort.<anonymous> (/opt/zigbee2mqtt/node_modules/zigbee-herdsman/src/adapter/z-stack/znp/znp.ts:146:28)
    at SerialPort._error (/opt/zigbee2mqtt/node_modules/@serialport/stream/dist/index.js:76:22)
    at /opt/zigbee2mqtt/node_modules/@serialport/stream/dist/index.js:112:18

**Comment by francisp2** [comment]
>cannot open /dev/serial/by-id/usb-ITead_Sonoff_Zigbee_3.0_USB_Dongle_Plus_2825e0cf8b29ec11a07e6d7840c9ce8d-if00-port0'

You did replace the port with the new serial/by-id of your new dongle ?

**Reply by 2dlight** [reply]
Where/ and how have I do it? 
Only edit in configuration.yaml?
Thanks

**Comment by francisp2** [comment]
ls /dev/serial/by-id
copy the id of your new dongle
stop zigbee2mqtt
replace in configuration.yaml
start zigbee2mqtt

**Reply by 2dlight** [reply]
Done and working.
Thanks!
